examples/byte-io/byte_io.bal

```diff
@@ -34,9 +34,9 @@ public function main() {
     string srcPath = "./files/ballerina.jpg";
     string dstPath = "./files/ballerinaCopy.jpg";
     // Initializes the readable byte channel.
-    io:ReadableByteChannel srcCh = io:openReadableFile(srcPath);
+    io:ReadableByteChannel srcCh = checkpanic io:openReadableFile(srcPath);
```

Comment:
Shall we handle error here, instead of using `checkpanic`? Ideally, we shouldn't promote `checkpanic` in examples. Can you please fix others as well.

Reply:
Fixed as suggested.